taxi/forms.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from taxi.models import Driver, Car
+from django.contrib.auth import get_user_model
+
+
+class DriverCreationForm(UserCreationForm):
+  class Meta(UserCreationForm.Meta):
+    model = Driver
+    fields = UserCreationForm.Meta.fields + (
+      "first_name",
+      "last_name",
+      "license_number", 
+    )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+  class Meta:
+      model = Driver
+      fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+   drivers = forms.ModelMultipleChoiceField(
+      queryset=get_user_model().objects.all(),
```

Comment:
The `queryset` for `drivers` in `CarForm` should be filtered to only include users who are drivers. Consider using `Driver.objects.all()` instead of `get_user_model().objects.all()` to ensure that only drivers are included in the queryset.